README.md

```diff
@@ -280,11 +280,13 @@ This changes the _frontend_ setting - the same as if you click your initials in
 Parameters:
 
 - `theme` - the name of the theme to set
+- `dark` - true to enable dark mode, false to enable light mode, null to automatically switch between the two. Only applicable to themes that support dark mode.
```

Comment:
Better to use an explicit value for automatic switching.

```yaml
dark: auto
```

Also, please change the description to "automatically follow your browser setting" or something. It was a bit unclear to me what this meant until I tested it.

Reply:
Done.